src/psyclone/tests/psyir/frontend/fparser2_test.py

```diff
@@ -423,6 +427,125 @@ def test_process_declarations_intent(f2008_parser):
         Symbol.Access.READWRITE
 
 
+def test_process_declarations_kind_new_param(f2008_parser):
+    ''' Test that process_declarations handles variables declared with
+    an explicit KIND parameter that has not already been declared. '''
```

Comment:
Please put ''' on a newline with a blank line inbetween. Similarly for other new tests.

Reply:
OK. I don't like it but I can live with it :-)